interactions/client/bot.py

```diff
@@ -1092,6 +1097,122 @@ def reload(
         self.remove(name, package)
         return self.load(name, package, *args, **kwargs)
 
+    async def get(
+        self,
+        obj: Union[Channel, Guild, Member, Message, Role, User],
+        *,
+        cache: bool = False,
+        channel_id: Optional[int] = MISSING,
```

Comment:
Instead of all these ID kwargs, how about just one?

Reply:
Oh I see. Guild ex. requires two values

Reply:
I don't like the clutter of all these keywords. For `Member`, `Message`, and `Role`, how about we expect a tuple or something?
```py
bot.get(Message, (guild_id, message_id))
```

Reply:
Do you think that's easier for the user?

Reply:
Well, it's certainly cleaner and prevents confusion, in my opinion

Reply:
Definitely. I'd like others' input as well, including fl0w

Reply:
Personally I think it would work better with kwargs than a single argument. If it were a tuple and you need to give two IDs, do they need to always be in the same order so we are able to correctly parse them? `(guild_id, message_id)` VS `(message_id, guild_id)`

Reply:
That was my thought

Reply:
I was about to suggest the same. If typehinting is an issue, then typing overloads can be used to do so in a pretty reasonable way.

Reply:
> Personally I think it would work better with kwargs than a single argument. If it were a tuple and you need to give two IDs, do they need to always be in the same order so we are able to correctly parse them? `(guild_id, message_id)` VS `(message_id, guild_id)`

I think a hierarchical process should be considered. You need the guild to get the message so (guild, message)

Reply:
> > Personally I think it would work better with kwargs than a single argument. If it were a tuple and you need to give two IDs, do they need to always be in the same order so we are able to correctly parse them? `(guild_id, message_id)` VS `(message_id, guild_id)`
> 
> I think a hierarchical process should be considered. You need the guild to get the message so (guild, message)

Honestly, I don't think it's *that* major - most people will put the guild ID before the message one, but if they don't... well, it's unorthodox, but it doesn't matter as long as they provide both.

Reply:
Seems easier to implement (and not less clean) then to use kwargs, don't you think?

Reply:
> Seems easier to implement (and not less clean) then to use kwargs, don't you think?

...really? Tuples are ugly in this case, and would be annoying to typehint (you would still have to use overloads to typehint per object). I'd argue it would take the same amount of time to typehint both for not much benefit.